### PR TITLE
adding tokenized version of Wagerr.

### PR DESCRIPTION
**Pull request by frndxyz**
initial request to add ERC20 and BEP20 tokenize version of Wagerr.

### CODE REVIEW

**Comment by Iamdeadlyz** [comment]
We don't have write-access to your PR. Please fix the checksum issue of the smartchain asset folder by following this:
- Rename the folder to `0xdBf8265B1d5244A13424f13977723AcF5395eAB2-x`, commit, and push
- Rename it again to `0xdBf8265B1d5244A13424f13977723AcF5395eAB2`, commit, then push

File: blockchains/smartchain/assets/0xdbf8265b1d5244a13424f13977723acf5395eab2/info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Wagerr",
+  "website": "https://www.wagerr.com",
+  "description": "BEP20 tokenized version of wagerr",
+  "explorer": "https://bscscan.com/token/0xdbf8265b1d5244a13424f13977723acf5395eab2",
+  "type": "BEP20",
+  "symbol": "WGR",
+  "decimals": 18,
+  "status": "active",
+  "id": "0xdbf8265b1d5244a13424f13977723acf5395eab2"
```
Comment (Iamdeadlyz) [inline blockchains/smartchain/assets/0xdbf8265b1d5244a13424f13977723acf5395eab2/info.json:10]: Change this to `"id": "0xdBf8265B1d5244A13424f13977723AcF5395eAB2"`

**Review by Iamdeadlyz — CHANGES_REQUESTED** [review]

File: blockchains/smartchain/assets/0xdbf8265b1d5244a13424f13977723acf5395eab2/info.json (hunk 1 above)
Comment (frndxyz) [reply blockchains/smartchain/assets/0xdbf8265b1d5244a13424f13977723acf5395eab2/info.json]: done

**Comment by frndxyz** [comment]
> We don't have write-access to your PR. Please fix the checksum issue of the smartchain asset folder by following this:
> 
> * Rename the folder to `0xdBf8265B1d5244A13424f13977723AcF5395eAB2-x`, commit, and push
> * Rename it again to `0xdBf8265B1d5244A13424f13977723AcF5395eAB2`, commit, then push

fixed this issue.

**Comment by frndxyz** [comment]
The Wagerr blockchain has been operating since 2018 and our chain activity can be seen on explorer.wagerr.com , we have recently launched wrapped tokens on eth and BSC. Your assistance in listing us on the trustwallet is greatly appreciated.

**Comment by frndxyz** [comment]
hi @Iamdeadlyz , @catenocrypt ,

any update on above listing?

**Comment by catenocrypt** [comment]
One of the 4 files (the last in this list) still has wrong folder, please rename that one too:

image

**Comment by frndxyz** [comment]
> One of the 4 files (the last in this list) still has wrong folder, please rename that one too:
> 
> image

requested changes has applied. thanks...

**Pull request merged**